backend/oasst_backend/prompt_repository.py

```diff
@@ -673,7 +673,8 @@ def fetch_message(self, message_id: UUID, fail_if_missing: bool = True) -> Optio
         messages = self._add_user_emojis_all(qry)
         message = messages[0] if messages else None
 
-        message = self.db.query(Message).filter(Message.id == message_id).one_or_none()
+        message = self.db.query(Message).filter(Message.id == message_id, not_(Message.deleted)).one_or_none()
```

Comment:
Oof, i can't approve for this.  You'll need to get the backend team to review this (or make it a separate PR)

Reply:
Understood. @andreaskoepf, please could you take a look?

Reply:
FetchMessage is a low level function that needs to be able to pull every message out of the db. I will make changes for deleted messages later if that should be necessary. Please restore this file for now.